### ISSUE TITLE
Improve article 2101.11533

### ISSUE THREAD

**Issue by luckywood** [dginev/ar5iv#49]
the conversion fails for unknown reasons
perhaps missing "addtokomafont" macros

**Comment by dginev** [comment]
Indeed, an [error cascade](https://ar5iv.org/log/2101.11533) of 100 errors leads latexml to quit prematurely as it misunderstood too much. As you have seen in the conversion log `\addtokomafont` is in part responsible, so this may relate to our 

latexml issue [915](https://github.com/brucemiller/LaTeXML/issues/915) for adding koma-script support.

**Comment by dginev** [comment]
The article is now mostly recovered, but we still need koma-script support to reach full success.
